bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/UniqueConfig.scala

```diff
@@ -0,0 +1,112 @@
+package jsky.app.ot.editor.seq
+
+import edu.gemini.spModel.config2.{Config, ConfigSequence}
+import edu.gemini.spModel.gemini.acqcam.InstAcqCam
+import edu.gemini.spModel.gemini.flamingos2.Flamingos2
+import edu.gemini.spModel.gemini.gmos.{GmosNorthType, GmosSouthType, InstGmosNorth, InstGmosSouth}
+import edu.gemini.spModel.gemini.gnirs.GNIRSConstants
+import edu.gemini.spModel.gemini.gsaoi.Gsaoi
+import edu.gemini.spModel.gemini.michelle.{InstMichelle, MichelleParams}
+import edu.gemini.spModel.gemini.nifs.InstNIFS
+import edu.gemini.spModel.gemini.niri.{InstNIRI, Niri}
+import edu.gemini.spModel.gemini.trecs.{InstTReCS, TReCSParams}
+import jsky.app.ot.editor.seq.Keys._
+
+
+/**
+ * Unique configurations represent sets of observes in a sequence which are done with the same
+ * instrument configuration and exposure times.
+ */
+case class UniqueConfig(count: Int, labels: String, config: Config) {
```

Comment:
I think it's fine to not try and glom additional ITC functionality into the existing `UniqueConfig` class.  We should probably rename this one though to avoid confusion.  Maybe `ItcUniqueConfig` or something.

The `labels` refer to data labels?  We have a type for that, `DatasetLabel` that you might be able to use instead?